pilz_control/include/pilz_control/pilz_joint_trajectory_controller_impl.h

```diff
@@ -133,7 +144,13 @@ bool PilzJointTrajectoryController<SegmentImpl, HardwareInterface>::
 updateTrajectoryCommand(const JointTrajectoryConstPtr& msg, RealtimeGoalHandlePtr gh, std::string* error_string)
 {
   std::lock_guard<std::mutex> lock(sync_mutex_);
-  return active_update_strategy_(msg, gh, error_string);
+  if(active_mode_ == Mode::HOLD)
+  {
+    return updateStrategyWhileHolding(msg, gh, error_string);
+  }
+
+  // The default case
+  return updateStrategyDefault(msg, gh, error_string);
```

Comment:
This is just the call of the base class method. I would remove `updateStrategyDefault()`, too.

Reply:
`updateStrategyWhileHolding` and `updateStrategyDefault` are one-liners. I'd suggest to inline them here directly (making the warning and the base class function call explicit).